docs/reference/release-notes/highlights.asciidoc

```diff
@@ -17,6 +17,31 @@ Other versions:
 | {ref-bare}/7.1/release-highlights-7.1.0.html[7.1]
 | {ref-bare}/7.0/release-highlights-7.0.0.html[7.0]
 
+// tag::notable-highlights[]
+[discrete]
+[[add-composable-index-templates]]
+=== Add composable index templates
+
+Index templates are an easy, repeatable way to configure mappings, index
+settings, and aliases for new indices. However, in previous versions, you had to
+define these configurations directly in the template. Managing multiple
+templates often meant copying similar configurations across templates.
+
+In 7.8, we added a more modular version of index templates called composable
+index templates. You can still define configurations directly in these
+templates. However, composable index templates can also contain component
+templates. Also added in 7.8, component templates are reusable configurations
+for mappings, index settings, and aliases. With a component template, you can
+define a configuration once and reuse it across multiple index templates. If you
+later need to change the configuration, you only need to change its component
+template.
+
+Composable index templates replace the previous version of index templates,
+which are now deprecated.
```

Comment:
Think it's worth mentioning that composable templates take precendence here?

Reply:
Thanks @dakrone. I'll add a sentence to note that.